libs/dh/metering-point/feature-search/src/lib/dh-search.component.ts

```diff
@@ -73,28 +75,60 @@ import { dhMeteringPointIdValidator } from './dh-metering-point.validator';
           }
         </watt-text-field>
       </div>
+
+      @if (meteringPointNotFound()) {
+        <watt-empty-state size="small" icon="custom-no-results" [title]="t('noResultFound')" />
+      }
     </vater-stack>
   `,
 })
 export class DhSearchComponent {
   private router = inject(Router);
   private doesMeteringPointExist = lazyQuery(DoesMeteringPointExistDocument);
 
-  searchControl = new FormControl('', [dhMeteringPointIdValidator()]);
+  searchControl = new FormControl('', {
+    validators: [dhMeteringPointIdValidator()],
+    nonNullable: true,
+  });
+
+  private seachControlChange = toSignal(this.searchControl.valueChanges);
+
+  meteringPointId = input<string>();
+  meteringPointNotFound = signal(false);
+
+  constructor() {
+    effect(() => {
+      const maybeMeteringPointId = this.meteringPointId();
+
+      if (maybeMeteringPointId) {
```

Comment:
Issue: if maybeMeteringPointId is something, meteringPointNotFound is set to true, is it not the other way around?